cypress/integration/student_cannot_see_deactivated_offerings.spec.js

```diff
@@ -0,0 +1,36 @@
+import C from '../support/constants.js';
+import * as TeacherHelper from '../support/helpers/teacherHelper';
+import * as StudentHelper from '../support/helpers/studentHelper';
+
+const CLASS_WORD = C.CLASS_WORD;
+const CLASS_NAME = 'Class '+ CLASS_WORD;
+const ASSIGNMENT_NAME = 'AP_AutomatedTestActivity1';
```

Comment:
This activity is renamed to Cypress_AutomatedTestActivity1_AP , can you please change it here?